examples/practice-typescript/src/components/Input/index.tsx

```diff
@@ -0,0 +1,20 @@
+import '../../styles/component.scss';
+
+interface InputProps {
+  typeInput: string;
+  styleClass: 'search';
```

Comment:
Pls recheck how to use an interface in TS. This is an object types but has the value.
https://www.typescriptlang.org/docs/handbook/2/objects.html